src/dataset/resource.rs

```diff
@@ -32,6 +73,10 @@ impl fmt::Display for Type {
             Self::Pdf => "PDF",
             Self::Csv => "CSV",
             Self::JsonLd => "JSON-LD",
+            Self::Xml => "XML",
+            Self::Image => "Image",
+            Self::Archive => "Packed Archive",
```

Comment:
Personally, I would prefer "Compressed archive" and "Plain text" here. Meaning no title case and compressed because it is more common than packed AFAIK.

Reply:
Changed in code